python/ray/utils.py

```diff
@@ -271,7 +274,16 @@ def get_cuda_visible_devices():
     if gpu_ids_str == "":
         return []
 
-    return [int(i) for i in gpu_ids_str.split(",")]
+    try:
+        gpu_ids = [int(i) for i in gpu_ids_str.split(",")]
+        return gpu_ids
+    except ValueError:
+        # SLURM sets CUDA_VISIBLE_DEVICES to NoDevFiles.
```

Comment:
Can we explicitly handle NoDevFiles instead of try catch

Reply:
done